gluten-core/src/main/scala/io/glutenproject/extension/ColumnarOverrides.scala

```diff
@@ -283,6 +283,10 @@ case class TransformPreOverrides(isAdaptiveContext: Boolean)
   }
 
   def replaceWithTransformerPlan(plan: SparkPlan): SparkPlan = {
+    if (plan.isInstanceOf[TransformSupport]) {
```

Comment:
Use `GlutenPlan` to cover more kinds of replaced plan?

Reply:
you're right. PR updated

Reply:
hi @PHILO-HE ,  unfortunately changing TransfromSupport to GlutenPlan will cause tests fail in velox. (see attached file). Since it takes time for me to setup velox dev env, I would suggested to go with TransformSupport, and open another issue to follow your advise.

Reply:
NP. Thanks!